vhdbuilder/packer/components.json

```diff
@@ -35,7 +35,8 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.4.44_hotfix"
+        "v1.4.44_hotfix",
+        "v1.5.3"
```

Comment:
its only this file now for linux? I thought there was additional junk that had to be touched but maybe I'm getting out of date?

Reply:
I think it's this and vhdbuilder/packer/install-dependencies.sh